validator/keymanager/remote-web3signer/types/requests.go

```diff
@@ -88,6 +90,34 @@ func GetAggregateAndProofSignRequest(request *validatorpb.SignRequest, genesisVa
 	}, nil
 }
 
+// GetAggregateAndProofV2SignRequest maps the request for signing type AGGREGATE_AND_PROOF_V2 on Electra changes.
+func GetAggregateAndProofV2SignRequest(v int, request *validatorpb.SignRequest, genesisValidatorsRoot []byte) (*AggregateAndProofV2SignRequest, error) {
```

Comment:
This function is not test covered.

Reply:
wait it is covered in TestKeymanager_Sign